plugins/extractors/couchdb/couchdb.go

```diff
@@ -0,0 +1,208 @@
+package couchdb
+
+import (
+	"context"
+	_ "embed" // used to print the embedded assets
+	"fmt"
+	"reflect"
+
+	_ "github.com/go-kivik/couchdb"
+	"github.com/go-kivik/kivik"
+	"github.com/odpf/meteor/models"
+	"github.com/odpf/meteor/models/odpf/assets"
+	"github.com/odpf/meteor/models/odpf/assets/common"
+	"github.com/odpf/meteor/models/odpf/assets/facets"
+	"github.com/odpf/meteor/plugins"
+	"github.com/odpf/meteor/registry"
+	"github.com/odpf/meteor/utils"
+	"github.com/odpf/salt/log"
+)
+
+//go:embed README.md
+var summary string
+
+var defaultDBList = []string{
+	"_global_changes",
+	"_replicator",
+	"_users",
+}
+
+// Config hold the set of configuration for the extractor
+type Config struct {
+	UserID   string `mapstructure:"user_id" validate:"required"`
+	Password string `mapstructure:"password" validate:"required"`
+	Host     string `mapstructure:"host" validate:"required"`
+}
+
+var sampleConfig = `
+ host: localhost:5984
+ user_id: admin
+ password: couchdb`
+
+// Extractor manages the extraction of data from MySQL
```

Comment:
Sorry for sounding picky, but I noticed that the comments still reflect `MySQL` as opposed to `CouchDB` while I was browsing the PR.

Similar occurrence at line 93.

Reply:
yup, missed changing that